src/styles/index.css

```diff
@@ -0,0 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Outfit:wght@100;300;400;600&display=swap');
+
+body {
+  margin: 0;
+  font-family: 'Outfit', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url("../../images/sunset-sky2.avif");
```

Comment:
In your app, you have a grey-ish block on the right size. It took me a minute to work out what was causing it but eventually I found out it was the background image you gave to body. When you give a background image to something (or if you add an image using `<img/>` ), the image will just have it's default size so it might not fit the way you want. In your case, the sunset image you used didn't fit al the way to the edge of the screen (for my screen at least). To fix this I added:
`background-size: cover;`
This makes sure the background image covers all the available space within it's container